ReactNativeClient/lib/components/screens/note.js

```diff
@@ -556,7 +560,15 @@ class NoteScreenComponent extends BaseScreenComponent {
 		const resourceTag = Resource.markdownTag(resource);
 
 		const newNote = Object.assign({}, this.state.note);
-		newNote.body += `\n${resourceTag}`;
+
+		if (this.state.mode == 'edit' && !Setting.value('editor.beta') && !!this.selection) {
```

Comment:
What's the issue with using this along with the beta editor?

Reply:
At the moment I see no point in trying. See:
* https://github.com/laurent22/joplin/issues/2939
* https://github.com/laurent22/joplin/issues/3036

Reply:
And it will probably requrie a different approach, I've never checked tbh.